extern/sector-storage/tarutil/systar.go

```diff
@@ -36,6 +36,7 @@ func ExtractTar(body io.Reader, dir string) error {
 			return xerrors.Errorf("creating file %s: %w", filepath.Join(dir, header.Name), err)
 		}
 
+		// XXX: Should we limit the amount of data we can write here?
```

Comment:
Do we trust the source here? `gosec` warns about potential TAR bombs.

Reply:
Yep, those are coming from cache transfers between workers, which ideally are setup on a separate local network, and use JWT auth